doc/contribute.rst

```diff
@@ -197,6 +197,10 @@ Kansha uses semantic HTML5 for the UI. Avoid presentation specific markup and in
 
 Python code should comply with PEP8. That requirement may be relaxed when it is difficult or impossible to follow, e.g. in views with many context managers (``with … :``).
 
+Since we use service injection, **all optional parameters to functions/classes must be explicitly passed as keyword arguments**, unless they are services.
+
```

Comment:
Maybe it was not  well formulated. « Pass optional parameters as explicit keyword arguments instead of positional arguments ». That's for safety. It is already considered a best practice in plain python to avoid ambiguity. It feels even more so in the context of service injection as we omit the passed services completely.